packages/docs/components/status.md

```diff
@@ -203,110 +249,115 @@ Status is intended to be both succinct and easily understood; limit label and de
-    </dt> 
+      Propulsion systems
+    </dt>
     <dd class="ods-status--value">
-      Service degredation
+      Check engine
     </dd>
   </dl>
   ```
 </figure>
 
-### Caution
+### Danger
 
 <figure class="docs-example">
   <div class="docs-example--rendered">
     <dl class="ods-status is-ods-status-danger">
       <dt class="ods-status--label">
-        Server status
-      </dt> 
+        Propulsion systems
+      </dt>
       <dd class="ods-status--value">
-        Service disruption
+        Warp core disruption
       </dd>
     </dl>
   </div>
 
   ```html
   <dl class="ods-status is-ods-status-danger">
     <dt class="ods-status--label">
-      Server status
-    </dt> 
+      Propulsion systems
+    </dt>
     <dd class="ods-status--value">
-      Service disruption
+      Warp core disruption
     </dd>
   </dl>
   ```
 </figure>
 
-## Accessibility
+## Hiding labels
+
+<Description>
 
-In addition to including copy for assistive technologies, consider whether your Status may change while the user is on the page.
+Labels may be hidden by applying the `.is-ods-status-label-hidden` class.
 
-If the current state of Status may change asynchronously while a user is visiting the page, utilize the `role="status"` attribute to ensure that assistive technologies correctly indicate this change.
+Even if the label is hidden, it must be populated to ensure appropriate context for users of assistive technology.
 
-<strong>Note:</strong> This attribute must be present <em>before</em> the change occurs.
+</Description>
 
 <figure class="docs-example">
   <div class="docs-example--rendered">
-    <dl class="ods-status is-ods-status-danger" role="status">
+    <dl class="ods-status is-ods-status-success is-ods-status-label-hidden">
       <dt class="ods-status--label">
-        Server status
+        Warp drive status
       </dt>
       <dd class="ods-status--value">
-        Service disruption
+        Warp drive engaged
       </dd>
     </dl>
   </div>
 
   ```html
-    <dl class="ods-status is-ods-status-danger" role="status">
-      <dt class="ods-status--label">
-        Server status
-      </dt>
-      <dd class="ods-status--value">
-        Service disruption
-      </dd>
-    </dl>
+  <dl class="ods-status is-ods-status-success is-ods-status-label-hidden">
+    <dt class="ods-status--label">
+      Warp drive status
+    </dt>
+    <dd class="ods-status--value">
+      Warp drive engaged
+    </dd>
+  </dl>
   ```
 </figure>
 
+## Accessibility
 
-### Assistive labels
+<Description>
 
-Labels may be hidden by applying the `.is-ods-status-label-hidden` class. If a label is not visually present, ensure that appropriate context is communicated by the Status itself and close proximity to the content that it’s supporting.
+If the current state of Status may change asynchronously while a user is visiting the page, utilize the `role="status"` attribute to ensure that assistive technologies correctly indicate this change.
```

Comment:
I would document this as a separate variant, I get that there are accessibility callouts here -- but documenting it as a child of accessibility can make this example feel difficult to find.

Reply:
I had that in my first pass, but it felt weird because it didn't match Overview - I think parity there is important.

We could add it to Overview as a variant, but It'll look identical to the other variants and have no guidelines other than the ones in the HTML/CSS tab.  🤔

Reply:
Took the same approach as I did with Hidden Labels - not couched as a variant, but also isn't couched as an Accessibility issue. Just dev notes, really.

Reply:
This LGTM!